src/plugins/api/score.h

```diff
@@ -203,10 +203,10 @@ class Score : public Ms::PluginAPI::ScoreElement
 //      Q_INVOKABLE void updateRepeatList(bool expandRepeats) { score()->updateRepeatList(); } // TODO: needed?
 
     /// \cond MS_INTERNAL
-    int nmeasures() const { return score()->nmeasures(); }
-    int npages() const { return score()->npages(); }
-    int nstaves() const { return score()->nstaves(); }
-    int ntracks() const { return score()->ntracks(); }
+    size_t nmeasures() const { return score()->nmeasures(); }
+    size_t npages() const { return score()->npages(); }
+    size_t nstaves() const { return score()->nstaves(); }
+    size_t ntracks() const { return score()->ntracks(); }
```

Comment:
... but changing to `size_t` here, would that work?

Reply:
It does build at least

Reply:
and without new warnings, as far as I can tell

Reply:
Actually with `int` futher up, it does generate 4 new warnings here

Reply:
I think, better keep `int`, and do like `int ntracks() const { return static_cast<int>(score()->ntracks()); }`

Reply:
But only here in this file, right?

Reply:
Done

Reply:
> But only here in this file, right?

right, only for Qml Api